packages/cbioportal-clinical-timeline/src/Timeline.tsx

```diff
@@ -203,14 +250,31 @@ const Timeline: React.FunctionComponent<ITimelineProps> = observer(function({
         zoomSelectBox: useRef(null),
         zoomSelectBoxMask: useRef(null),
         cursorText: useRef(null),
+        hoverStyleTag: useRef(null),
+        timelineViewPort: useRef(null),
     };
 
+    const memoizedHoverCallback = useCallback(
+        (e: React.MouseEvent) => {
+            hoverCallback(e, refs.hoverStyleTag);
+        },
+        [refs.hoverStyleTag]
+    );
+
     // on mount, there will be no element to measure, so we need to do this on equivalent
     // of componentDidMount
     useEffect(() => {
         setTimeout(() => {
-            store.updateViewPortWidth();
+            store.viewPortWidth = jQuery(
```

Comment:
why?

Reply:
because i don't want jquery stuff in store.  i could stuff this in a function but not too much point

Reply:
I was confused because you left updateViewPortWidth in the store, but now I understand you just forgot to delete it